server/controllers/comment.controller.js

```diff
@@ -2,10 +2,7 @@ import dotenv from 'dotenv';
 import db from '../models';
 import { Response } from '../utils';
 
-dotenv.config();
-
-
-const { comments, User } = db;
+const { comments, User, commentlikes } = db;
```

Comment:
Why is there no consistency in the database table names?

Reply:
Will edit it